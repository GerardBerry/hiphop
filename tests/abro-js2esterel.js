var js2esterel = require("../js2esterel.js");
var abro = require("./abro.js");

console.log(abro.prg.esterel_code());
