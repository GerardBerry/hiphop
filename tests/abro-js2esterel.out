module ABRO:
input R, A, B;
output O;
loop
    abort
        [     await A
          ||
              await B ];
        emit O;
        halt
    when R
end loop
end module
